docs/image/README.md

```diff
@@ -0,0 +1,16 @@
+# Docs Test
+this container helps to test the documentation before you push to github
+
+## How-to
+
+```
+$ docker build -t local/deis-docs .
+$ docker run --rm -t -i -v `pwd`/../../../deis/deis:/deis -p 8000:8000 local/deis-docs
+```
```

Comment:
It would be awesome to have this command available in `docs/Makefile`. Something like `make serve-preview`, which starts the server and prints that the doc preview is now available via `http://localhost:8000`.